webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -16,7 +23,7 @@
         <img
           v-if="contribution"
           class="contribution-image"
-          :src="contribution.image | proxyApiUrl"
+          :src="contribution.image || proxyApiUrl"
```

Comment:
```suggestion
          :src="contribution.image | proxyApiUrl"
```

Reply:
this single pipe character `|` runs `contribution.image` through a filter at `webapp/plugins/vue-filters.js`
which takes in a url(contribution.image) and if it starts with a `/` meaning it's a relative url, then it replaces `/` with `/api/` so that it proxies our backend for the image
double pipes `||` mean `or`, maybe you know all this and it's an unintended change, but it's not one we want, I guess.